courses/course_planning/electives.md

```diff
@@ -0,0 +1,205 @@
+---
+id: electives
+title: Electives To Take
+sidebar_label: Electives To Take
+slug: /course_planning/electives
+---
+
+_Last updated on 2022/07/29_
+
+Most CS programs require students to take some Arts/Languages/Social Science classes. The Honours Computer Science program, for example, requires three of such courses (at least one in Arts/Languages and one in Social Science).
+
+**The bolded courses are considered to be bird courses by other CSS members who have taken them.**
+
+:::warning
+
+This list gives you an overview about classes that you may want to take to satisfy your requirements. However, you should consult an advisor first or look at your [UWinsite Students page](/wiki/resources/guides/uwinsite_guide#academic-progress) to see whether they can satisfy the requirements of your specific program.
+
+:::
+
+## Arts Courses
+
+-   **GART-1500: Effective Writing I** - this class is easy if you feel comfortable with writing and grammar. There are a lot of independent readings, weekly assignments, quizzes, and paragraph-writing. However, the classes are short (50 minutes per week). Make sure to follow the rubric and talk to your GA/TA whenever you need help!
+
+<details>
```

Comment:
Is there a non-html way to represent this?

Also, the different folding menus are a little misleading -- "course codes" makes me expect ABCD-XYZW, not just the subject. (ABCD is called the subject of the course code).

Alternate titles: `Course Options` and `Qualifying Subjects`

Reply:
Thanks for the suggestions! 

I really like the alternative titles, but I think I'll use `Qualifying Subject Codes` just to make it clearer that the contents are part of a course code.

I don't think there is a non-html way to represent the folding menus, but I could replace it with a table/list.

Reply:
Re. The non-html stuffs- according to docusaurus docs there is no way (they actually tell you to do it the way you did, lol)

https://docusaurus.io/docs/markdown-features#details

I think the `<li>` stuff can be replaced with markdown lists (`- this is a lost element`)

Reply:
I'll try it! Markdown does this weird thing where the list sometimes doesn't work and everything sticks to one line, but it may have to do with indentations of the elements

Reply:
Update: the Markdown list element works as long as it is at the same indentation as the <details> tag!

Reply:
That's great :) Today I Learned! 🚀 🚀